src/main/java/io/javalin/plugin/openapi/JavalinOpenApi.kt

```diff
@@ -48,8 +56,16 @@ object JavalinOpenApi {
                 updateComponents {
                     applyMetaInfoList(options.handlerMetaInfoList, options)
                 }
+
                 updatePaths {
                     applyMetaInfoList(options.handlerMetaInfoList, options)
+                    options.overridenPaths?.forEach { (path, documentation) ->
+                        applyMetaInfoList(options.handlerMetaInfoList.filter {
+                            PathParser(path.first).matches(it.path) && it.httpMethod.name == path.second.name
+                        }.ifEmpty {
+                            listOf(HandlerMetaInfo(HandlerType.valueOf(path.second.name), path.first, Handler { }, emptySet()))
+                        }.map { it.copy(handler = documented(documentation, Handler { })) }, options)
+                    }
```

Comment:
I think we should move the implementation into a new function, to not break the abstraction level of this method. For example `applyOverriddenPaths`.

Reply:
val overriddenHandlerMetaInfoList = overridePaths(options.handlerMetaInfoList, options.overridenPaths); // Override or add the documentation of the configured paths
updateComponents {
    applyMetaInfoList(overriddenHandlerMetaInfoList, options)
}

updatePaths {
    applyMetaInfoList(overriddenHandlerMetaInfoList, options)
    ensureDefaultResponse()
}

Reply:
It should now working as intended